org.eclipse.winery.bpmn4tosca.converter.tobpel/src/main/java/org/eclipse/winery/bpmn2bpel/model/OrGatewayMerge.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2017 ZTE Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.winery.bpmn2bpel.model;
+
+public class OrGatewayMerge extends Gateway {

```

Comment:
'OrGatewayMerge' is the end of exclusive gateway and 'OrGatewaySplit' is the start of exclusie gateway. The reason for these names is that in the source code there are two nodes 'AndGatewaySplit' and 'AndGatewayMerge'.  I believe these two nodes are the start and end of parallel gateway respectively, so I use 'OrGateway' to stands for exclusive gateway.